lib/LLVMPasses/ARCEntryPointBuilder.h

```diff
@@ -193,8 +193,7 @@ class ARCEntryPointBuilder {
     // intrinsics are atomic today.
     if (I->getIntrinsicID() != llvm::Intrinsic::not_intrinsic)
       return false;
-    return (I->getCalledFunction()->getName().find("nonatomic") !=
-            llvm::StringRef::npos);
+    return (I->getCalledFunction()->getName().contains("nonatomic"));
```

Comment:
It looks like you are handling StringRef and Set-like types. Really this should be two separate commits. I am fine if it is in the same PR. But it should be explicit since a StringRef is /not/ a set like type and contains in the context of a StringRef means something else.

Reply:
Done!